docs/program/extend/modular-resources/_index.md

```diff
@@ -24,9 +24,9 @@ Two key concepts exist across all Viam resources (both built-in and modular) to
 
 ### APIs
```

Comment:
At the top of this page can you add `nolist: true` and then manually add the examples as cards in a next steps section at the bottom as well?

Reply:
yep! That lgtm 
It duplicates cards a bit but I'm fine with that